src/spec/verifiable-payid-protocol-NC.md

```diff
@@ -0,0 +1,381 @@
+---
+coding: utf-8
+
+title: Self-Sovereign Verifiable PayID
```

Comment:
Let's rename the filename to `verifiable-payid-protocol-self-sov.md ` or something that doesn't have "NC"?